.github/workflows/flowzone.yml

```diff
@@ -301,33 +301,37 @@ jobs:
       versionbot: ${{ steps.versionbot_checkout.outcome }}
 
     steps:
-      - name: Checkout versioned source
-        id: versionbot_checkout
-        continue-on-error: true
-        uses: actions/checkout@v3
-        with:
-          ref: versionbot/pr/${{ github.event.pull_request.number }}
-          fetch-depth: 2
-          token: ${{ secrets.FLOWZONE_TOKEN }}
-
-      - name: Assert that versioned source is up-to-date
-        if: ${{ steps.versionbot_checkout.outcome == 'success' }}
-        run: git merge-base --is-ancestor ${{ github.event.pull_request.head.sha }} HEAD
+      # - name: Checkout versioned source
+      #   id: versionbot_checkout
+      #   continue-on-error: true
+      #   uses: actions/checkout@v3
+      #   with:
+      #     ref: versionbot/pr/${{ github.event.pull_request.number }}
+      #     fetch-depth: 2
+      #     token: ${{ secrets.FLOWZONE_TOKEN }}
```

Comment:
No need to keep the commented lines around :)

Reply:
I'm still testing. I'll clean up before I finalize the PR.